disk/tests/fixtures/blkid_cache_file

```diff
@@ -0,0 +1 @@
+<device DEVNO="0xca01" TIME="1578466057.746380" LABEL="MYLABEL" UUID="3364ebc8-7e32-4850-8391-76f1de97f453" TYPE="ext4" PARTUUID="9583252c-01">/dev/sda1</device>
```

Comment:
(nit) I mean previously to have one file with multiple lines :)

Reply:
I've tried to do that but the problem is that we need to change the psutil mock to return multiple devices, not just metrics for /dev/sda1 as it currently does. Hence, I can only have one device named /dev/sda1 in this file. Otherwise, I'll need to change the psutil mocks and potentially everything and it felt that this refactoring would increase the scope of the PR by quite a bit. I'm okay to do it if you prefer it this way though, let me know.

Reply:
As discussed, I've made `blkid_cache_file` a multi line file to ensure that the parsing is working as intended